.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "helm-charts"]
 	path = helm-charts
 	url = https://github.com/otterize/helm-charts.git
+[submodule "api"]
```

Comment:
Change the GHA `checkout` action so that it also checks out submodules during build